exporter/signalfxexporter/config.go

```diff
@@ -94,7 +94,7 @@ func (cfg *Config) getOptionsFromConfig() (*exporterOptions, error) {
 
 	var metricTranslator *translation.MetricTranslator
 	if cfg.SendCompatibleMetrics {
-		metricTranslator, err = translation.NewMetricTranslator(cfg.TranslationRules)
+		metricTranslator, err = translation.NewMetricTranslator(cfg.TranslationRules, 1)
```

Comment:
Does it mean we have 1 second ttl? it's very small. the minimum metrics interval we have by default is 10s :) 
I think default should be 1 hour.
And we probably need to expose it as a configuration in case if user wants to have metrics with interval of more than hour.

Reply:
Oh yeah, forgot to update this value. Will make it configurable.